samples/cpp/speech_sample/main.cpp

```diff
@@ -345,8 +345,10 @@ int main(int argc, char* argv[]) {
         // -----------------------------------------------------------------------------------------------------
         // --------------------------- Step 5. Do inference --------------------------------------------------------
         std::vector<std::vector<uint8_t>> ptrUtterances;
-        std::vector<std::vector<uint8_t>> vectorPtrScores((outputs.size() == 0) ? 1 : outputs.size());
-        std::vector<uint16_t> numScoresPerOutput((outputs.size() == 0) ? 1 : outputs.size());
+        std::vector<std::vector<uint8_t>> vectorPtrScores((outputs.size() == 0) ? executableNet.outputs().size()
```

Comment:
Looks like it is not necessary to define the size of those vectors here it will reduce code size